test/resources/schema.json

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
can we name the file something like `submit-return-des-schema-v-1-0-0` so it's more obvious what it is? If we end up using  more schemas it'll help with that. Also, it's nice to know the version of the spec the schema came from

Reply:
also, when you run the service can the `countries.json`  still be found since you moved it to the test directory?

Reply:
schema.json renamed to submit-return-des-schema-v-1-0-0.json

Reply:
yes, all tests are passing